lib/instrumentation/@elastic/elasticsearch.js

```diff
@@ -55,9 +64,17 @@ function queryParser(params) {
     queryParam = params.querystring
   }
   // let body or bulkBody override querystring, as some requests have both
-  if (typeof params.body === 'object' && Object.keys(params.body).length > 0) {
+  if (
+    typeof params.body === 'object' &&
```

Comment:
we have a function in `lib/config/merge-server-config.js` isSimpleObject.  We could move that to utility and use that there and in these two places so we can have some abstract helper going forward

Reply:
up to you. just wanted to call it out that this is a valuable check

Reply:
Moved into util and both elastic and merge-server config now reference it there.